### PR TITLE
[animations] added test for focus image

### PR DESCRIPTION
**Pull request by arteevraina**
Added widget test for `FocusImage`.

ref: #399

### CODE REVIEW

**Comment by arteevraina** [comment]
@RedBrogdon @domesticmouse Please review this PR and suggest changes :)

File: animations/test/misc/focus_image_test.dart [hunk 1]
```diff
@@ -0,0 +1,56 @@
+// Copyright 2019 The Flutter team. All rights reserved.
```
Comment (RedBrogdon) [inline animations/test/misc/focus_image_test.dart:1]: nit: new file should have a 2021 year.

File: animations/test/misc/focus_image_test.dart [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:animations/src/misc/focus_image.dart';
```
Comment (RedBrogdon) [inline animations/test/misc/focus_image_test.dart:7]: nit: These imports should be sorted.

File: animations/test/misc/focus_image_test.dart [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      home: FocusImageDemo(),
+    );
+
+void main() {
+  group('FocusImage Tests', () {
+    testWidgets('Size of ink well changes on tap', (tester) async {
+      await tester.pumpWidget(createFocusImageScreen());
+
+      // Get the initial inkwell.
+      final initialInkwell = tester.firstWidget(find.byType(InkWell));
```
Comment (RedBrogdon) [inline animations/test/misc/focus_image_test.dart:19]: It's probably not an issue, but this test is using two different methods to find the first InkWell widget (`firstWidget` and `.at(0)`).

For consistency and readability, convert the searches on line 19 and 29 to use:

tester.widget(find.byType(InkWell).at(0))

**Review by RedBrogdon — CHANGES_REQUESTED** [review]
Just a few nits.

File: animations/test/misc/focus_image_test.dart (hunk 3 above)
Comment (arteevraina) [reply animations/test/misc/focus_image_test.dart:19]: sure thing!

**Comment by arteevraina** [comment]
@RedBrogdon I have made the changes.

**Review by RedBrogdon — APPROVED** [review]
Thanks for improving our testing!

**Pull request merged**